services/app-web/src/components/HealthPlanPackageTable/HealthPlanPackageTable.tsx

```diff
@@ -0,0 +1,331 @@
+        }),
+        columnHelper.accessor('submissionType', {
+            id: 'submissionType',
+            header: 'Submission type',
+            cell: (info) => <span>{info.getValue()}</span>,
+            meta: {
+                dataTestID: 'submission-type',
+            },
+            filterFn: `arrIncludesSome`,
+        }),
+        columnHelper.accessor('programs', {
+            header: 'Programs',
+            cell: (info) =>
+                info.getValue().map((program) => {
+                    return (
+                        <Tag
+                            data-testid="program-tag"
+                            key={program.id}
+                            className={`radius-pill ${styles.programTag}`}
+                        >
+                            {program.name}
+                        </Tag>
+                    )
+                }),
+            meta: {
+                dataTestID: 'submission-programs',
+            },
+        }),
+        columnHelper.accessor('submittedAt', {
+            header: 'Submission date',
+            cell: (info) =>
+                info.getValue()
+                    ? dayjs(info.getValue()).format('MM/DD/YYYY')
+                    : '',
+            meta: {
+                dataTestID: 'submission-date',
+            },
+        }),
+        columnHelper.accessor('updatedAt', {
+            header: 'Last updated',
+            cell: (info) =>
+                info.getValue()
+                    ? dayjs(info.getValue()).format('MM/DD/YYYY')
+                    : '',
+            meta: {
+                dataTestID: 'submission-last-updated',
+            },
+        }),
+        columnHelper.accessor('status', {
+            header: 'Status',
+            cell: (info) => <StatusTag status={info.getValue()} />,
+            meta: {
+                dataTestID: 'submission-status',
+            },
+        }),
+    ]
+
+    const reactTable = useReactTable({
+        data: tableData.sort((a, b) =>
+            a['updatedAt'] > b['updatedAt'] ? -1 : 1
+        ),
+        columns: tableColumns,
+        getCoreRowModel: getCoreRowModel(),
+        state: {
+            columnFilters,
+            columnVisibility: {
+                stateName: userType !== 'StateUser',
+                submissionType: userType !== 'StateUser',
+            },
+        },
+        onColumnFiltersChange: setColumnFilters,
+        getFacetedUniqueValues: getFacetedUniqueValues(),
+        getFilteredRowModel: getFilteredRowModel(),
+        getSortedRowModel: getSortedRowModel(),
+    })
+
+    const stateColumn = reactTable.getColumn('stateName')
+    const submissionTypeColumn = reactTable.getColumn('submissionType')
+
+    // Filter options based on table data instead of static list of options.
+    // const stateFilterOptions: FilterOptionType[] = Array.from(stateColumn.getFacetedUniqueValues().keys()).map(state => ({
+    //     value: state,
+    //     label: state
+    // }))
```

Comment:
I still need to touch base with design about how we want to display the options. The commented out code here will only display unique options contained in the tableData instead of a static list.